### ISSUE TITLE
Curate and suggest post with related topic in forum

### ISSUE THREAD

**Issue by ziesski** [Submitty/Submitty#10353]
**What problem are you trying to solve with Submitty**

Current Submitty forum takes time to search all good related topics.

**Describe the way you'd like to solve this problem**

When we type the title for new post, we want submitty to populate curated suggestions "this posts seem related, have you seen them?" 

This was discussion from Barb's interactive visualization class, suggested by student

**Comment by CCWizma99** [comment]
Hello @ziesski. I think you are speaking about Submitty's discussion forum, right? And the new functionality will be to auto-suggest suitable posts like we see in a web browser that suggest us content while we type in search bar...?

**Comment by ziesski** [comment]
> ut Submitty's discussion forum, right? And the new functionality will be to auto-suggest suitable posts like we see in a web browser that suggest us content while we type in search bar...?

yes, basically we want submitty forum to check if there is any existing post on duplicate topic.
